src/ain/node/node-client.ts

```diff
@@ -84,7 +77,7 @@ export class NodeClient {
 
   // UTXO
   get utxoFee(): number {
-    return this.chain === Chain.MAIN ? 0.00000132 : 0.0000222;
+    return this.chain === 'main' ? 0.00000132 : 0.0000222;
```

Comment:
String in environment variable is 'mainnet'.

Reply:
how did it work before then? cause enum option was 'main'

Reply:
Yes, that's somewhat confusing. The getInfo call returns 'main' whereas the Whale API client requires 'mainnet' 🤔 that's why in the config we have 'mainnet'.

Reply:
nice... :)